skfp/fingerprints/lingo.py

```diff
@@ -0,0 +1,183 @@
+import hashlib
+import re
+from collections import defaultdict
+from collections.abc import Sequence
+from numbers import Integral
+from typing import Optional, Union
+
+import numpy as np
+from rdkit.Chem import Mol
+from scipy.sparse import csr_array
+from sklearn.utils import Interval
+
+from skfp.fingerprints.base import FingerprintTransformer
+from skfp.validators import ensure_smiles
+
+
+class LingoFingerprint(FingerprintTransformer):
+    """
+    Lingo fingerprint.
+
+    The Lingo fingerprint [1]_ is a hashed fingerprint based on cdk that counts/flags the occurrences of
```

Comment:
Remove `based on cdk` - we reimplement this entirely, and this is based basically only on the original paper

Reply:
Maybe `that check the occurrences of substrings...`? Count version is covered by parameter description

Reply:
I would more explicitly underline here that while original paper uses raw dictionaries, we return constant-length, hashed fingerprints